samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ConfirmPublishAssessmentListener.java

```diff
@@ -317,12 +319,18 @@ else if (time instanceof java.lang.Boolean) {
       g = (GradebookExternalAssessmentService) SpringBeanLocator.getInstance().
             getBean("org.sakaiproject.service.gradebook.GradebookExternalAssessmentService");
     }
+    String toGradebook = assessmentSettings.getToDefaultGradebook();
     try{
-	if (assessmentSettings.getToDefaultGradebook() && gbsHelper.isAssignmentDefined(assessmentSettings.getTitle(), g)){
-        String gbConflict_err= ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.AssessmentSettingsMessages" , "gbConflict_error");
-        context.addMessage(null,new FacesMessage(gbConflict_err));
-        error=true;
-      }
+    	if (toGradebook!=null && StringUtils.equals(toGradebook, toDefaultGradebook) &&
```

Comment:
I don't think you need to test for `null` here, as `StringUtils.equals()` is null safe. Same on L:329